Components/Queues/Drivers/AsyncQueueDriverInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers;
+
+use Smartbox\Integration\FrameworkBundle\Core\Consumers\AbstractAsyncConsumer;
+use Smartbox\Integration\FrameworkBundle\Core\Consumers\AbstractConsumer;
```

Comment:
In this interface there is no use of these 2 classes, so these lines can be deleted

Reply:
Done.